### ISSUE TITLE
glance_printer doesn't work with the latest glance

### ISSUE THREAD

**Issue by mooreryan** [bcpeinhardt/glance_printer#2]
The glance_printer package does not currently work with the latest version of glance.  I figured I would take a look at trying to get it working with the latest version, but when building and running tests from the github repo I get a few errors (like in-exhaustive case expressions).  I assume that this is because glance itself accepts more than the gleam compiler.  That leads to my question of, when developing, is there a way you run the tests such that the compiler doesn't give errors?

**Comment by bcpeinhardt** [comment]
TBH I'm not sure if I'll continue to update glance_printer. The majority of times people have used it they've been advised in the discord that it's not the appropriate tool for their use case haha. 
What are you currently trying to accomplish with glance_printer?

**Comment by mooreryan** [comment]
Some sort of codegen (probably what most others were attempting as well)--reading in some gleam code, generating AST with glance, making some additions, refactors, etc, and printing out with glance_printer.

**Comment by bcpeinhardt** [comment]
I've published an updated version (v1.2.0) which should work with v0.11.0 of glance for you :) I did it quite hastily so let me know if you have issues.

**Comment by CrowdHailer** [comment]
I have a project on glance printer and am discovering the same issue. I am using glance 3.0.0.
What was the advised approach

**Comment by CrowdHailer** [comment]
I'm going to stick with glance_printer. I use it in https://hex.pm/packages/oas_generator which see's some use.
I'm going to build an mcp_generator tool along the same principle

**Comment by bcpeinhardt** [comment]
@CrowdHailer I'm no longer using glance_printer for much of anything, and it seems like it will need to track glance pretty regularly. Would you be interested in taking over as the repo maintainer? I'm not sure who else is using it regularly.

**Comment by CrowdHailer** [comment]
Yeah I'm happy to take over. Looks like the PR I need is already open here

**Comment by bcpeinhardt** [comment]
Awesome! Thank you so much! Yep I just haven't had time to review test and merge. FYI for me to transfer ownership of the repo over to you, you'll have to delete your existing fork.

**Comment by CrowdHailer** [comment]
Done.
Can you also transfer (or add me) to the hex package.

**Comment by bcpeinhardt** [comment]
Yes 😎 
Edit: Done
